### DISCUSSION TITLE
No way to control audio output format?

### DISCUSSION THREAD

**Discussion by elkay** [FelixWaweru/elevenlabs-node#54] [Q&A]
Default audio quality is 128khz. Higher subscription levels allow up to 192khz and also PCM lossless. Official API controls these via query string parameters (why they aren't JSON properties is a question for the official team).

**Comment by FelixWaweru** [comment]
Hi @elkay 
Thank you for sharing this issue. This has been added to the feature list and will be available in the next release

- #55
